content/800-data-platform/100-accelerate/580-local-development.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Local development'
+metaTitle: 'Accelerate: Local development'
+metaDescription: 'Learn how to use Prisma Accelerate in a development environment.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma Accelerate is designed to scale database connections efficiently in production environments. Prisma Accelerate also provides a global cache to reduce the load on your database and reduce query response time. To leverage Prisma Accelerate, it is essential to utilize a publicly accessible database.
+
+In development environments, you may want to use a local database to minimize expenses. Furthermore, you may consider extending Prisma Client with the Accelerate client extension once so that you can use a local database in development and a hosted database with Accelerate’s connection pooling and caching enabled. This eliminates the need for conditional logic to switch clients between development and production.
+
+This guide will explain how to use Prisma Accelerate client extension in a development environment with a local database.
+
+</TopBlock>
+
+## Using Prisma Accelerate client extension in development and production
+
+<br />
+<img src="images/accelerate-in-dev.png" alt="" />
+
+Accelerate does not work with a local database. However, in a development environment, you can still use Prisma Client with the Accelerate client extension. This setup will not include Accelerate's connection pooling and caching features.
+
+The following steps outline how to use Prisma ORM and Prisma Accelerate with a local PostgreSQL database.
+
+1. Update the `DATABASE_URL` environment variable with your local database's connection string:
+
+   ```.env
+   DATABASE_URL="postgres://username:password@127.0.0.1:5432/localdb"
+   ```
+
+2. Generate a Prisma Client:
+
+   ```bash
+   npx prisma generate
+   ```
+
+   > Note: The `--no-engine` flag should only be used in preview and production environments. The command generates Prisma Client artifacts without a [Query Engine](/concepts/components/prisma-engines) file, which requires an Accelerate connection string.
+
+3. Set up Prisma Accelerate to your project:
+
+   ```typescript
+   import { PrismaClient } from '@prisma/client'
```

Comment:
In usual Accelerate deployments this require `@prisma/client/edge`, right? 
So you still need different code for local development?

Reply:
Do you mean the following statement at the beginning of the text?

> This eliminates the need for conditional logic to switch clients between development and production.

When using a regular Node.js runtime, this statement is true.

In the case of an edge function, as of now, you'd have to use the Prisma edge client, which requires the use of Accelerate. Hence the original statement does not apply to this edge case. 

Do you feel that the copy edit below sounds better/more specific

> This eliminates the need for conditional logic to switch clients between development and production (except for edge functions).